src/platforms/java/migration.mdx

```diff
@@ -42,6 +42,16 @@ val hints = mutableMapOf<String, Any>("myHint" to "myStringHint")
 Sentry.captureException(RuntimeException("exception"), hints)
 ```
 
+* `SentryOptions#sendClientReports` is now enabled by default. To disable it, use the code snippet below:
+
+```properties
+send-client-reports=false
+```
```

Comment:
Let's say Im a new dev joining a project that already uses Sentry.
If I see this snippet above, maybe I not sure what to do with it.
Would be nice instead of `use the code snippet below` something like 'set the following property in the file X`.

Reply:
Hmm there's multiple ways here right? `sentry.properties` where you just put `send-client-reports=false`, `application.properties` for Spring where you put `sentry.send-client-reports=false` and the init block where you can use `options.setSendClientReports(false);`. Then there's also the Kotlin version and instead of an `application.properties` file you can also use `application.yml` for Spring. Would it be easier to the option to the docs and just link it here? Seems like documenting this option has become more important now that it allows you to avoid the self hosted upgrade.

Reply:
I guess we just need to document the most common one, people will know that you can do it in different ways since every other property can be configured whether in the properties files, SDK init via options or yml.
Adding all the possibilities here is just boilerplate IMO.
@adinauer If you wish to document all the possibilities and link to them here, what do you prefer?

Reply:
Documenting and linking here seems like the cleaner approach considering support may have to help users getting around the self hosted upgrade. Simply linking to the docs would probably also be easier then. Can send a PR if you wish.

Reply:
@adinauer yes please, let's merge as it is then and you can submit a PR on top of this one so people at least know that this is a breaking change (its already published), still need the approval, thanks.